Dinero.Tools.Difference/Dinero.Tools.Difference.Core/DataParsers/HandelsbankenDataParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dinero.Tools.Difference.Core.Interfaces;
+using Dinero.Tools.Difference.Core.Models;
+using FileHelpers;
+
+namespace Dinero.Tools.Difference.Core.DataParsers
+{
+    public class HandelsbankenDataParser : IDataParser
+    {
+        public IEnumerable<EntryModel> Parse(string data)
+        {
+            var engine      = new FileHelperEngine<SparNordEntry>();
+            var parseResult = engine.ReadString(data);
+            var result      = new List<EntryModel>();
+
+            for (var index = 0; index < parseResult.Length; index++)
+            {
+                var shbEntry = parseResult[index];
+                var entryModel = new EntryModel()
+                {
+                    Index   = index,
+                    Amount  = shbEntry.Amount,
+                    Date    = shbEntry.Date,
+                    Text    = shbEntry.Text,
+                    Saldo   = shbEntry.Saldo
+                };
+                result.Add(entryModel);
+            }
+
+            return result;
+        }
+
+        [DelimitedRecord(";"), IgnoreEmptyLines(), IgnoreFirst()]
+        private class SparNordEntry
```

Comment:
@larsbaunwall Guess you missed renaming this one? I know it's private and all, but the name is miss leading.